node/resolvers/search/index.ts

```diff
@@ -58,9 +58,15 @@ enum CrossSellingInput {
   accessories = 'accessories',
 }
 
+enum CrossSellingGroupByInput {
+  PRODUCT = 'PRODUCT',
+  NONE = 'NONE',
+}
```

Comment:
why did you chose to use a enum instead of using a prop like `groupByProduct` that could just be true or false? do we intend to allow other types of grouping in the future?

Reply:
The idea to always use enum is because we can easily add other kinds of group types if needed in the future, if we go for boolean and there's another type of group that we discover, we will need to create a new prop